### PR TITLE
PHPCS: Rest of the packages

### PR DESCRIPTION
**Pull request by kraftbj**
Makes the rest of the packages directory PHPCS compliant except for:

* Queue.php in the sync package. Will be made so as part of #13806 so deferring to that PR.
* analyzer/ -- A few more files are compliant, but there's still a lot left. This package is not actually used by Jetpack or any other project that I'm aware of. Initially, I marked to ignore the whole thing, but on second thought, I'd rather that we eventually get it compliant or remove it from the repo. Since it isn't used anywhere, it would not be potentially blocking for a future WP.com commit.

#### Changes proposed in this Pull Request:
* Various PHPCS related updates.

#### Is this a new feature or does it add/remove features to an existing part of Jetpack?
* Supports the composer on wp.com work.

#### Testing instructions:
* n/a

#### Proposed changelog entry for your changes:
* n/a

**One request: If there needs to be any changes to this beyond what someone can suggest, may I ask that it be added in a new PR targeting this branch? Due to the high LOC count, I'd like to land this quickly so tweaks in a separate PR would help support that.**

### CODE REVIEW

**Review by zinigor — CHANGES_REQUESTED** [review]
This is great, thank you for doing it! I have found some minor issues, could you take a look please?

File: packages/analyzer/scripts/example.php [hunk 1]
```diff
@@ -1,59 +1,78 @@
 <?php
+/**
+ * Example usage script.
+ *
+ * @package automattic/jetpack-anaylizer
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply packages/analyzer/scripts/example.php]: ef1646a41

File: packages/analyzer/scripts/jetpack-svn.php [hunk 2]
```diff
@@ -9,32 +9,42 @@
  * two are optional, and probably a bit gratuitous :)
  *
  * `php jetpack-svn.php path/to/your/plugin/code`
+ *
+ * @package automattic/jetpack-anaylzer
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply packages/analyzer/scripts/jetpack-svn.php]: ef1646a41

File: packages/options/legacy/class.jetpack-options.php [hunk 3]
```diff
@@ -396,9 +434,10 @@ static function update_raw_option( $name, $value, $autoload = false ) {
 		if ( ! $updated_num ) {
 			$updated_num = $wpdb->query(
 				$wpdb->prepare(
-					"INSERT IGNORE INTO $wpdb->options ( option_name, option_value, autoload ) VALUES ( %s, %s, '$autoload_value' )",
+					"INSERT IGNORE INTO $wpdb->options ( option_name, option_value, autoload ) VALUES ( %s, %s, %s )",
 					$name,
-					$serialized_value
+					$serialized_value,
+					$autoload_value
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply packages/options/legacy/class.jetpack-options.php]: It is using `$autoload_value` though, which is set to either `yes` or `no` via L411 `$autoload_value = $autoload ? 'yes' : 'no';`

I would expect this to be fine be loaded in the way I changed it?

File: packages/analyzer/scripts/example.php [hunk 4]
```diff
@@ -1,59 +1,78 @@
...
 $invocations->scan( $external_base_path . '/example-external.php' );
-// $invocations->scan( '/Users/dan/workspace/a8c/some-repo', array( '.git', '.gitmodules', 'assets' ) );
-// $invocations->output();
+// $invocations->scan( '/Users/dan/workspace/a8c/some-repo', array( '.git', '.gitmodules', 'assets' ) ); // .
+// $invocations->output(); // .
 
 echo "\n*** List of dependencies\n";
 $dependencies = new Automattic\Jetpack\Analyzer\Dependencies();
-$dependencies->generate( $invocations, $jp74_declarations, $repo_path );
+$dependencies->generate( $invocations, $jp74_declarations, $data_path );
```
[the remark this answers is not in the document]
Comment (kraftbj) [reply packages/analyzer/scripts/example.php]: ef1646a41

File: packages/options/legacy/class.jetpack-options.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (zinigor) [reply packages/options/legacy/class.jetpack-options.php]: Yes, thank you, I get it now, I missed that part of the code before.

**Review by zinigor — APPROVED** [review]

**Comment by jeherve** [comment]
That PR title... It's perfect :)

d6d

**Review by jeherve — APPROVED** [review]
This looks good. Merging. Thanks a lot for that!

Marking this for 8.0 for now.

**Pull request merged**